frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -466,6 +466,7 @@ export type ResourceLinkProps = {
   title?: string;
   dataTest?: string;
   onClick?: () => void;
+  truncate?: boolean;
```

Comment:
Since this is a public API, do we need to update the tsdoc?

Reply:
Ah, thanks. It was part of #11639 which hasn't merged yet. @zherman0 FYI

Reply:
Ok, I took @zherman0 tsdoc and added `truncated` prop